packages/features/ee/teams/components/AddNewTeamMembers.tsx

```diff
@@ -33,11 +34,21 @@ type FormValues = {
 const AddNewTeamMembers = () => {
   const searchParams = useCompatSearchParams();
   const session = useSession();
+  const telemetry = useTelemetry();
+
   const teamId = searchParams?.get("id") ? Number(searchParams.get("id")) : -1;
   const teamQuery = trpc.viewer.teams.get.useQuery(
     { teamId },
     { enabled: session.status === "authenticated" }
   );
+
+  useEffect(() => {
```

Comment:
Can we track directly from server side? I don't think this should be a client side event.

Reply:
Update: we can but we need to upgrade next-collect. Will do in a follow up.